src/dlgNotepad.cpp

```diff
@@ -73,22 +78,39 @@ void dlgNotepad::save()
     mNeedToSave = false;
 }
 
-void dlgNotepad::restore()
+void dlgNotepad::restoreFile(const QString& fName, const bool useUtf8Encoding)
 {
-    QString fileName = mudlet::getMudletPath(mudlet::profileDataItemPath, mpHost->getName(), QStringLiteral("notes.txt"));
     QFile file;
-    file.setFileName(fileName);
+    file.setFileName(fName);
```

Comment:
I don't like this readability

Reply:
I wanted to ensure I was using a different name inside method to that outside - but can change that - would `fn` be the right way to go (or not)?

Reply:
IMHO that would be even worse. I like to read and understand variable names. Not sure why fileName didn't seem to suffice anymore.

Reply:
:thinking: When I refactor a block of code into a separate method I do not like having the same variables in the called method as those that were in the original code - because it sometimes makes you think you are using global variables /class members instead of function arguments - whilst syntax highlighting can prevent that in the IDE it does not help if quoting code extracts elsewhere...